utils/src/main/scala/cromwell/pipeline/utils/configs/LogInfoUtils.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.pipeline.utils.configs
+
+import cromwell.pipeline.utils._
+import cromwell.pipeline.utils.configs.SecretData._
+import play.api.libs.functional.syntax.{ toFunctionalBuilderOps, unlift }
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json.{ Json, Writes, __ }
+
+trait LogInfoUtils {
+  def getConfigMessage: String
+}
+
+object LogInfoUtils {
+
+  def getConfigMessage(applicationConfig: ApplicationConfig): String = {
+
+    def createMessageHeader: String = {
```

Comment:
Do you need to calculate this every time? I think val is better here

Reply:
Great point, ill fix it, thanks